ontology/uco/observable/observable.ttl

```diff
@@ -8524,6 +9043,28 @@ observable:X509V3ExtensionsFacet
 	sh:targetClass observable:X509V3ExtensionsFacet ;
 	.
 
+observable:XiaomiFacet
```

Comment:
This Facet needs a name correction.

(Sorry if this is a repeat, I remember typing this once already...)